MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+"""Bazel module for bazel-annotate-buildkite-plugin."""
+
+module(
+    name = "bazel_annotate_buildkite_plugin",
+    version = "1.0.0",
+)
+
+# No external dependencies needed - using only built-in Bazel rules
```

Comment:
We'd have dependencies on protobuf.

```
bazel_dep(name = "protobuf", version = "31.1")
bazel_dep(name = "rules_proto", version = "7.1.0")
bazel_dep(name = "rules_python", version = "1.5.1")

protobuf = use_extension("@protobuf//:extensions.bzl", "protobuf")
protobuf.toolchain(name = "protobuf")
```

and then updating build rules in `BUILD.bazel` probably